l10n/en/firefox/browsers/mobile/ios-summarizer.ftl

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.springfield.moz.works/landing/ios-summarizer/
+
+# Note for localizers: "shake things up" is an English idiom meaning to cause change, disrupt the usual way,
+# or bring excitement. Please translate with an equivalent expression in your language
+# that conveys energy, freshness, or innovation, rather than a literal "shake" action.
+mobile-ios-summarizer-page-title = Ready to shake things up?
+mobile-ios-summarizer-page-description =
+    Get instant, tailored summaries in { -brand-name-firefox } { -brand-name-ios }.
+    Shake your phone to cut through the clutter with recipe ingredients, sports highlights,
+    or article overviews in seconds.
+
+# Note for localizers: "shake things up" is an English idiom meaning to cause change, disrupt the usual way,
+# or bring excitement. Please translate with an equivalent expression in your language
+# that conveys energy, freshness, or innovation, rather than a literal "shake" action.
+mobile-ios-summarizer-hero-title = Ready to shake things up?
```

Comment:
BTW `-hero-title` has been dangling since using `-page-title` for that and IIUC should also be removed?

```suggestion
```